README.md

```diff
@@ -231,8 +231,8 @@ If a given `Animal` has no children, its `child_names` list is empty.
 #### Constraints and Rules
 - `@fold` can only be applied to vertex fields, except the root vertex field.
 - May not exist at the same vertex field as `@recurse`, `@optional`, or `@output_source`.
-- Expanding vertex fields is not allowed within a scope marked `@fold`.
-- `@tag` and `@fold` may not be used within a scope marked `@fold`.
+- Only one vertex field may be expanded within a scope marked `@fold`. Note that this applies for all nested scopes within.
+- `@tag`, `@recurse`, `@optional`, `@output_source` and `@fold` may not be used at any level within a scope marked `@fold`.
```

Comment:
`level` is not defined anywhere in the documentation -- let's try to stick to the existing language and terms for the sake of not confusing people. How about simply "anywhere within a scope marked @fold"?